### ISSUE TITLE
jc:  Error - date parser could not parse the input data.

### ISSUE THREAD

**Issue by paxsali** [kellyjonbrazil/jc#95]
```
$ date
Mon 04 Jan 2021 10:43:06 PM CET

$ pip3 show jc
Name: jc
Version: 1.14.0
Summary: Converts the output of popular command-line tools and file-types to JSON.
Home-page: https://github.com/kellyjonbrazil/jc
Author: Kelly Brazil
Author-email: kellyjonbrazil@gmail.com
License: MIT
Location: /home/paxsali/.local/lib/python3.8/site-packages
Requires: xmltodict, Pygments, ruamel.yaml
Required-by:

$ jc date

jc:  Error - date parser could not parse the input data. Did you use the correct parser?
             For details use the -d or -dd option.
``` 

I don't see what seems to be the problem.

Runnning Ubuntu 20.04 LTS.

Any suggestions what's wrong?

**Comment by kellyjonbrazil** [comment]
Hi!

Thanks for reporting this issue. Could you post the output with the `-dd`?

Also, please post the output of the date command all by itself (without `jc`)

Finally - does doing this help?
```
$ LANG=C jc date
```
Thanks!

**Comment by kellyjonbrazil** [comment]
Ah never mind - it looks like I have what I need already. Thanks!

**Comment by kellyjonbrazil** [comment]
Looks like your output has `PM` in it where I haven't seen that before on other systems. Is that an environment or LANG setting? Either way, I can build a fix for that. Thanks!

**Comment by paxsali** [comment]
```
$ date --version
date (GNU coreutils) 8.30
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David MacKenzie.

$ echo $LANG
en_US.UTF-8

$ echo $LC_ALL
en_US.UTF-8
```

**Comment by kellyjonbrazil** [comment]
I'll spin up an Ubuntu 20.04 LTS instance to reproduce. In the mean time could you post the output of the `alias` command? I'm curious as to whether the distro added a `date` alias during install or if this is just the way the newest version of `date` works. I was testing with `date` version 8.22 on linux and an ancient version of `date` on BSD and macOS.

**Comment by kellyjonbrazil** [comment]
Ok - so I think I see what's going on here. It looks like there may be new behavior in the latest version of `date`

https://www.gnu.org/software/coreutils/manual/html_node/date-invocation.html#date-invocation

> Invoking date with no format argument is equivalent to invoking it with a default format that depends on the LC_TIME locale category. In the default C locale, this format is ‘'+%a %b %e %H:%M:%S %Z %Y'’, so the output looks like ‘Thu Mar  3 13:47:51 PST 2005’.

When I run `jc date` on a fresh install of Ubuntu 20.04 LTS it fails as you report, but if I preface the command with `LANG=C`, then it works:
```
$ LANG=C jc date
{"year": 2021, "month_num": 1, "day": 5, "hour": 0, "minute": 17, "second": 44, "month": "Jan", "weekday": "Tue", "weekday_num": 3, "timezone": "UTC"}
```
So I think what I will do is update the parser so it will recognize this new format and parse correctly since it looks like it is a default behavior for a popular distro.

**Comment by kellyjonbrazil** [comment]
I have a fix that I will merge into `jc` v1.14.1, but you can try it now by placing a file called `date.py` under `$HOME/.local/share/jc/jcparsers` with the code from here:

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/date.py

This will override the existing `date` parser with the fix.

**Comment by kellyjonbrazil** [comment]
Changes merged into v1.14.1, now available on pypi.
